frontend/lib/requests/feed.ts

```
@@ -4,6 +4,7 @@ import { Announcement, AuthorizedUser, Droplet } from "@/types";
 import qs from "qs";
 import { flattenAttributes } from "../utils";
 import { revalidatePath } from "next/cache";
+import { AnnouncementType } from "@/types";
```

ESLint

[error] 'AnnouncementType' is defined but never used.

(@typescript-eslint/no-unused-vars)